### DISCUSSION TITLE
How to disable images?

### DISCUSSION THREAD

**Discussion by FuadEfendi** [hbstack/.github#4] [Q&A]
Hi Razon,

How to disable page thumbnails and featured images? I cannot find,  https://github.com/hbstack/hb/tree/main/layouts/partials/hb/functions

My site articles don't use images:
https://fuadefendi.com/

Thanks,

**Comment by razonyang** [comment]
The thumbnails can not be disabled for now, will expose a parameter for this later.

And the carousel (featured image) can be disabled by removing the `featured` parameter from page's front matter.

**Comment by razonyang** [comment]
Hi, the thumbnails can be disabled now, available since `github.com/hbstack/blog` v0.8.0 (https://github.com/hbstack/blog/pull/75).

You'll need to upgrade that module via `hugo mod get github.com/hbstack/blog@v0.8.0`, and then set the `hb.blog.post_thumbnail` parameter as `false`.

https://github.com/hbstack/theme/blob/d6fd20adf76973a6979d4505ac0d09a8171456bb/config/_default/params.yaml#L24

**Comment by FuadEfendi** [comment]
Thank you Razon!

**Reply by razonyang** [reply]
Should be fixed too (docs doesn't have a homepage layout), you can post a screenshot if there are something I missed.

**Comment by razonyang** [comment]
FYI, the `hb.blog.post_thumbnail_default` and `hb.blog.post_thumbnail_placeholder` parameters are available since blog `v0.10.0`, to set the default thumbnail image or text. Maybe better than no image.

See https://hbstack.dev/en/docs/modules/blog/overview/#site-parameters
